repair/repair_description.hh

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (C) 2023-present ScyllaDB
```

Comment:
2024 :-)

Reply:
Also, sorry about the bike-shedding, but why did you decide to call this file repair_description.hh? A person looking at "ls" might think this is something completely different - maybe the description of ongoing repairs in nodetool or task list or I don't know what. Maybe it should be called repair_metrics.hh or repair_metric_names.hh or something?

Reply:
+1

Reply:
@nyh if we wait long enough, it will be 2025
file naming is bullet 3 on the decision list, I wish we had a poll options to get a majority decision.